common-libs/did-auth-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/affinidi-did-auth-lib",
-  "version": "2.2.0",
+  "version": "3.0.0",
```

Comment:
despite this library uses libraries that have to be update with major version, this library (did-auth-lib) doesn't use bloom-vault under the hood, so my opinion **to update it with a minor update**

Reply:
changed